### ISSUE TITLE
CSV export function gives Unicode error when the language is set to Dutch

### ISSUE THREAD

**Issue by Woseseltops** [Signbank/NGT-signbank#142]
Related to #117

**Comment by henrinie** [comment]
This is a pure guess, but could it be solved by changing the __str__ functions to __unicode__?

**Comment by Woseseltops** [comment]
The problem is that Dutch header names have some non-ASCII chars (for example 'oriëntatie'). I fixed that now by ignoring these trouble characters (so it will now say 'orintatie'), so we get an export again, but I foresee some problems:

* When importing this csv, the fieldname will not be recognized
* The bullet above assumes that the csv importer knows which language we're using, and assumes field labels in that language, which I don't think is the case. Or do we want language detection? Simply said: the importer now assumes and English csv.
* The various field values are also still in English; so we now have Dutch labels, but English values.

Shall we make things easy for ourselves, and force the csv to always be in English, regardless of the interface language?

**Comment by ocrasborn** [comment]
Yes, please!

**Comment by Woseseltops** [comment]
Hmm... this might be more complicated than I thought:

http://stackoverflow.com/questions/5258715/django-switching-for-a-block-of-code-switch-the-language-so-translations-are-d

**Comment by Woseseltops** [comment]
Done, the labels are now always English! I tested this Django functionality step by step in a separate view, and ported it over to the csv code once I verified i had a working solution. In the end, it was only three minor changes to the code, but it took a lot of googling and testing to get it right, as people don't do language switching in Django often, apparently.

On a separate note: while doing this task i started to get annoyed that I had to go to signbank.science.ru.nl/reload_signbank and then wait a few second all the time before I could see changes. I sidetracked a bit and created a script that monitors the repo, and automatically goes to reload_signbank when it detects file changes. Saves me some time and manual actions, and helps me remaining 'in the flow'.

**Comment by richardbank** [comment]
Not sure what is going on, but apparently the en_US translation list is used for NGT Signbank. An older csv file from SB contains a column headed 'Strong Hand'; when I try to import the changes, the column is skipped because it doesn't exist. When I export a new CSV from SB, this column is headed 'Dominant Hand Shape' (as a comparison, the column 'Weak Hand' is still labeled 'Weak Hand')
This also applies to the list of fields in the phonological description (of course, i think).

**Comment by Woseseltops** [comment]
Note to self for when I continue on this: the problem was that the Dutch Signbank thinks english = american english. When Richard was trying this, the translation was halfway, after refreshing the entire Dutch signbank uses American English. Fix: "en" != "en_US"

**Comment by Woseseltops** [comment]
I think I got this working again! 

The problem was that Signbank assumed that the 'en_US' language files should be used when the language was 'en'. I tried all kinds of crazy ideas to make Django stop assuming this, most notably adding a whole new language 'en_GB', but it lead to all kinds of weird behaviors because Signbank is not ready for sublocales (US vs GB). 

My final solution is so simple I can't believe it: Django was aware of the American version because I compiled it once (= creating a .mo file out of the .po file). If I simply delete that .mo file, we get the default text again! We don't track .mo files in Git, so it won't affect the American version.
